mods/player_api/api.lua

```diff
@@ -62,6 +62,16 @@ function player_api.set_model(player, model_name)
 	player_model[name] = model_name
 end
 
+function player_api.reset_boxes(player, model_name)
+	local model = models[model_name]
+	if model then
+		player:set_properties({
+			collisionbox = model.collisionbox or
+				{-0.3, 0.0, -0.3, 0.3, 1.7, 0.3},
```

Comment:
I'd like to have this table as a local constant in this mod. This way it could also be used in `player_api.set_model` to ease changing the defaults and to ensure that the same table is used in both locations.

Reply:
Ok will do.
And no i think it's bad practice to use the parameters of one use of an API within the API's functions.

Reply:
Done.